tools/memleak.py

```diff
@@ -223,12 +203,12 @@ def run_command_get_pid(command):
         sizes.delete(&pid);
 
         info.timestamp_ns = bpf_ktime_get_ns();
-        info.num_frames = grab_stack(ctx, &info) - 2;
```

Comment:
We were dropping two frames here, and I'm not exactly sure why. At least for userland stacks, these frames seem perfectly fine.